ansible/configs/ocp4-workshop/files/repos_template.j2

```diff
@@ -1,27 +1,28 @@
 [rhel-7-server-rpms]
 name=Red Hat Enterprise Linux 7
-baseurl={{own_repo_path}}/rhel-7-server-rpms
+baseurl={{own_repo_path}}/ocp/common/rhel-7-server-rpms
 enabled=1
 gpgcheck=0
 
 [rhel-7-server-rh-common-rpms]
 name=Red Hat Enterprise Linux 7 Common
-baseurl={{own_repo_path}}/rhel-7-server-rh-common-rpms
+baseurl={{own_repo_path}}/ocp/common/rhel-7-server-rh-common-rpms
```

Comment:
TODO: bring the usage of that path with @ fridim -- rationale is that two extra repos don't have the same structure.

Potential solutions:
- Having everyone updating their env_vars
- Creating the two extra repos via Ansible "repo" module 
- Adding new variables for accommodating more than one repo path

Reply:
Should be fixed in the `status-api-test` branch.